content/docs/api-reference/assets-api/index.md

```diff
@@ -13,4 +13,171 @@ weight: 101
 toc: true
 ---
 
+## Asset API Examples
+
+Create the [bearer_token](../../setup-and-administration/getting-access-tokens-using-client-secret) and store in a file in a secure local directory with 0600 permissions.
+
+Set the URL (for example):
+
+```bash
+export URL=https://synsation.1234-5678.nodes.archivist.jitsuin.io 
+```
+
+### Asset Record Creation
+
+Define the asset parameters and store in `/path/to/jsonfile`:
+
+```json
+{
+    "behaviours": ["RecordEvidence", "Attachments"],
+    "attributes": {
+        "arc_firmware_version": "1.0",
+        "arc_serial_number": "vtl-x4-07",
+        "arc_display_name": "tcl.ppj.003",
+        "arc_description": "Traffic flow control light at A603 North East",
+        "arc_home_location_identity": "locations/115340cf-f39e-4d43-a2ee-8017d672c6c6",
+        "arc_display_type": "Traffic light with violation camera",
+        "some_custom_attribute": "value",
+        "arc_attachments": [
+            {
+                "arc_display_name": "arc_primary_image",
+                "arc_attachment_identity": "blobs/87b1a84c-1c6f-442b-923e-a97516f4d275",
+                "arc_hash_alg": "SHA256",
+                "arc_hash_value": "246c316e2cd6971ce5c83a3e61f9880fa6e2f14ae2976ee03500eb282fd03a60"
+            }
+        ]
+    }
+}
+```
+
+Create the asset:
+
+```bash
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    $URL/archivist/v2/assets
+```
+
+The response is:
+
+```json
+{
+    "identity": "assets/3f5be24f-fd1b-40e2-af35-ec7c14c74d53",
+    "behaviours": [
+        "RecordEvidence",
+        "Attachments"
+    ],
+    "attributes": {
+        "arc_serial_number": "vtl-x4-07",
+        "arc_display_name": "tcl.ppj.003",
+        "arc_description": "Traffic flow control light at A603 North East",
+        "arc_home_location_identity": "locations/115340cf-f39e-4d43-a2ee-8017d672c6c6",
+        "arc_display_type": "Traffic light with violation camera",
+        "arc_firmware_version": "1.0",
+        "some_custom_attribute": "value",
+        "arc_attachments": [
+            {
+                "arc_display_name": "arc_primary_image",
+                "arc_attachment_identity": "blobs/87b1a84c-1c6f-442b-923e-a97516f4d275",
+                "arc_hash_alg": "SHA256",
+                "arc_hash_value": "246c316e2cd6971ce5c83a3e61f9880fa6e2f14ae2976ee03500eb282fd03a60"
+            }
+        ]
+    },
+    "confirmation_status": "PENDING",
+    "tracked": "TRACKED"
+}
+```
+
+### Asset Record Retrieval
+
+Asset records in Jitsuin Archivist are tokenized at creation time and referred to in all API calls and smart contracts throughout the system by a unique identity of the form:
```

Comment:
All instances of “Jitsuin Archivist” or “Archivist” need to be changed to RKVST.

This should be an easy search-and-replace but be careful not to change any of the URLs or resource identifiers which do still use ‘archivist’

Reply:
All should be updated, barring the API endpoints

Reply:
There's one left, but I'll address that in my PR for the tagline since it's in the same area